From: Jiang Xin <jiangxin@moon.ossxp.com>
Subject: [PATCH] Setting dialog for my hackings.

Setting dialog for my hackings, such as:

* Unicode output,
* Save extra attributes in other XML file.

Signed-off-by: Jiang Xin <jiangxin@moon.ossxp.com>

---
 freemind/freemind.properties                       |   13 ++++++-----
 .../freemind/preferences/layout/OptionPanel.java   |   22 ++++++++++++++++++++
 2 files changed, 29 insertions(+), 6 deletions(-)

diff --git a/freemind/freemind.properties b/freemind/freemind.properties
index fc7132f..93097ff 100644
--- a/freemind/freemind.properties
+++ b/freemind/freemind.properties
@@ -555,12 +555,13 @@ resources_signed_script_are_trusted=true
 icons.list=idea;help;yes;messagebox_warning;stop-sign;closed;info;button_ok;button_cancel;full-1;full-2;full-3;full-4;full-5;full-6;full-7;full-8;full-9;full-0;stop;prepare;go;back;forward;up;down;attach;ksmiletris;smiley-neutral;smiley-oh;smiley-angry;smily_bad;clanbomber;desktop_new;gohome;folder;korn;Mail;kmail;list;edit;kaddressbook;knotify;password;pencil;wizard;xmag;bell;bookmark;penguin;licq;freemind_butterfly;broken-line;calendar;clock;hourglass;launch;flag-black;flag-blue;flag-green;flag-orange;flag-pink;flag;flag-yellow;family;female1;female2;male1;male2;fema;group
 # not added: encrypted;decrypted;redo;
 
-# fc, 23.3.2009: Here, the default node font is taken as the default note's font, too.
-# possible values: true or false (or anything else, that is interpreted as "false", too).
-# This property waits for being integrated into the OptionPanel after the 0.9.0 release.
-resources_use_default_font_for_notes_too=true
-# fc, 23.3.2009: Here, a margin-top:0; is added to the style of each note causing 
-# a line spacing of zero above each paragraph.
+# worldhello, 25.5.2008:
+wh_nonascii_in_utf8=true
+wh_save_extra_attrs_in_aux_file=true
+wh_separate_attr_folded=true
+wh_separate_attr_created=true
+wh_separate_attr_modified=true
+
 # possible values: true or false (or anything else, that is interpreted as "false", too).
 # This property waits for being integrated into the OptionPanel after the 0.9.0 release.
 resources_use_margin_top_zero_for_notes=true
diff --git a/freemind/freemind/preferences/layout/OptionPanel.java b/freemind/freemind/preferences/layout/OptionPanel.java
index 19a6951..d7cedd5 100644
--- a/freemind/freemind/preferences/layout/OptionPanel.java
+++ b/freemind/freemind/preferences/layout/OptionPanel.java
@@ -1154,6 +1154,28 @@ public class OptionPanel implements TextTranslator {
         controls.add(new NumberProperty("undo_levels.tooltip", "undo_levels", 2,1000,1));
 
 		/***********************************************************************
+		 * WorldHello ****************************************************************
+		 */
+		controls.add(new NewTabProperty("wh_extension"));
+		controls.add(new SeparatorProperty("wh_nonascii_in_utf8"));
+		controls.add(new BooleanProperty(
+				"wh_nonascii_in_utf8.tooltip",
+				"wh_nonascii_in_utf8")); // true
+		controls.add(new SeparatorProperty("wh_mm_aux_file"));
+		controls.add(new BooleanProperty(
+				"wh_save_extra_attrs_in_aux_file.tooltip",
+				"wh_save_extra_attrs_in_aux_file")); // true
+		controls.add(new BooleanProperty(
+				"wh_separate_attr_folded",
+				"wh_separate_attr_folded")); // true
+		controls.add(new BooleanProperty(
+				"wh_separate_attr_created",
+				"wh_separate_attr_created")); // true
+		controls.add(new BooleanProperty(
+				"wh_separate_attr_modified",
+				"wh_separate_attr_modified")); // true
+
+		/***********************************************************************
 		 * Browser/external apps
 		 * ****************************************************************
 		 */
-- 
tg: (19169f1..) t/extensions_setting_dialog (depends on: tgmaster)
